Library/Formula/rabbitmq.rb

```diff
@@ -1,14 +1,12 @@
-# We'd really like this formula to be built from source so we can build
-# against a modern OpenSSL rather than the prehistoric system version.
-# Submit a PR if you're interested, Thanks!
```

Comment:
I don't *think* so. I was under the impression that using Homebrew's Erlang meant that the OpenSSL problem was solved.